core/templates/dev/head/pages/dashboard/Dashboard.js

```diff
@@ -205,7 +225,11 @@ oppia.controller('Dashboard', [
         } else {
           $scope.activeTab = 'myExplorations';
         }
-        $rootScope.loadingMessage = '';
+        if ($location.path() === '/back') {
```

Comment:
This is pretty hacky. Should "Loading" be set here? It seems weird to set it like this _after_ the data is fetched...